starport/cmd/network_campaign_account.go

```diff
@@ -0,0 +1,139 @@
+package starportcmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/tendermint/starport/starport/pkg/entrywriter"
+	"github.com/tendermint/starport/starport/services/network"
+	"github.com/tendermint/starport/starport/services/network/networktypes"
+)
+
+var (
+	campaignMainnetsAccSummaryHeader = []string{"Mainnet Account", "Shares"}
+	campaignVestingAccSummaryHeader  = []string{"Vesting Account", "Total Shares", "Vesting", "End Time"}
+)
+
+// NewNetworkCampaignAccount creates a new campaign account command that holds some other
+// sub commands related to account for a campaign.
+func NewNetworkCampaignAccount() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "account",
+		Short: "Handle campaign accounts",
+	}
+
+	c.AddCommand(
+		newNetworkCampaignAccountList(),
+	)
+	c.PersistentFlags().AddFlagSet(flagNetworkFrom())
```

Comment:
We don't this in query apis.

Reply:
We can't remove these flags now because the network command will not work. The network method requires both flags to use the helper methods. We can refactor it in another PR